diff --git a/.DS_Store b/.DS_Store
index 8bb5042..360f06b 100644
Binary files a/.DS_Store and b/.DS_Store differ
diff --git a/src_3/.DS_Store b/src_3/.DS_Store
index 5008ddf..25c5da8 100644
Binary files a/src_3/.DS_Store and b/src_3/.DS_Store differ
diff --git a/src_3/__pycache__/clean_data_seq2point.cpython-38.pyc b/src_3/__pycache__/clean_data_seq2point.cpython-38.pyc
index 646c77e..ba3aa9a 100644
Binary files a/src_3/__pycache__/clean_data_seq2point.cpython-38.pyc and b/src_3/__pycache__/clean_data_seq2point.cpython-38.pyc differ
diff --git a/src_3/__pycache__/config_file.cpython-38.pyc b/src_3/__pycache__/config_file.cpython-38.pyc
index 8aa73c9..c388bfa 100644
Binary files a/src_3/__pycache__/config_file.cpython-38.pyc and b/src_3/__pycache__/config_file.cpython-38.pyc differ
diff --git a/src_3/__pycache__/data_loaders.cpython-38.pyc b/src_3/__pycache__/data_loaders.cpython-38.pyc
index 691c1bf..2f14a70 100644
Binary files a/src_3/__pycache__/data_loaders.cpython-38.pyc and b/src_3/__pycache__/data_loaders.cpython-38.pyc differ
diff --git a/src_3/__pycache__/train_fed.cpython-38.pyc b/src_3/__pycache__/train_fed.cpython-38.pyc
index 83f6934..93837a4 100644
Binary files a/src_3/__pycache__/train_fed.cpython-38.pyc and b/src_3/__pycache__/train_fed.cpython-38.pyc differ
diff --git a/src_3/config_file.py b/src_3/config_file.py
index 9f93537..18fd2ab 100644
--- a/src_3/config_file.py
+++ b/src_3/config_file.py
@@ -1,4 +1,5 @@
 path = "/home/Alfredo/input/1min_real_sept_oct_nov_dec2019.csv"
+path_2 = "/Users/alfredorodriguez/Desktop/input/1min_real_sept_oct_nov2019.csv"
 
 def load_hyperparameters(appliance):
 
@@ -6,7 +7,7 @@ def load_hyperparameters(appliance):
         config = dict(
             appliance=str(appliance),
             epochs=2000,
-            batch_size=2500,
+            batch_size=128,
             learning_rate=3.3180899699083407e-04,#e-06,
             in_channels=1,
             out_channels=16,
diff --git a/src_3/data/MNIST/processed/test.pt b/src_3/data/MNIST/processed/test.pt
deleted file mode 100644
index 7379e51..0000000
Binary files a/src_3/data/MNIST/processed/test.pt and /dev/null differ
diff --git a/src_3/data/MNIST/processed/training.pt b/src_3/data/MNIST/processed/training.pt
deleted file mode 100644
index 12b8e06..0000000
Binary files a/src_3/data/MNIST/processed/training.pt and /dev/null differ
diff --git a/src_3/data/MNIST/raw/t10k-images-idx3-ubyte b/src_3/data/MNIST/raw/t10k-images-idx3-ubyte
deleted file mode 100644
index 1170b2c..0000000
Binary files a/src_3/data/MNIST/raw/t10k-images-idx3-ubyte and /dev/null differ
diff --git a/src_3/data/MNIST/raw/t10k-images-idx3-ubyte.gz b/src_3/data/MNIST/raw/t10k-images-idx3-ubyte.gz
deleted file mode 100644
index 5ace8ea..0000000
Binary files a/src_3/data/MNIST/raw/t10k-images-idx3-ubyte.gz and /dev/null differ
diff --git a/src_3/data/MNIST/raw/t10k-labels-idx1-ubyte b/src_3/data/MNIST/raw/t10k-labels-idx1-ubyte
deleted file mode 100644
index d1c3a97..0000000
Binary files a/src_3/data/MNIST/raw/t10k-labels-idx1-ubyte and /dev/null differ
diff --git a/src_3/data/MNIST/raw/t10k-labels-idx1-ubyte.gz b/src_3/data/MNIST/raw/t10k-labels-idx1-ubyte.gz
deleted file mode 100644
index a7e1415..0000000
Binary files a/src_3/data/MNIST/raw/t10k-labels-idx1-ubyte.gz and /dev/null differ
diff --git a/src_3/data/MNIST/raw/train-images-idx3-ubyte b/src_3/data/MNIST/raw/train-images-idx3-ubyte
deleted file mode 100644
index bbce276..0000000
Binary files a/src_3/data/MNIST/raw/train-images-idx3-ubyte and /dev/null differ
diff --git a/src_3/data/MNIST/raw/train-images-idx3-ubyte.gz b/src_3/data/MNIST/raw/train-images-idx3-ubyte.gz
deleted file mode 100644
index b50e4b6..0000000
Binary files a/src_3/data/MNIST/raw/train-images-idx3-ubyte.gz and /dev/null differ
diff --git a/src_3/data/MNIST/raw/train-labels-idx1-ubyte b/src_3/data/MNIST/raw/train-labels-idx1-ubyte
deleted file mode 100644
index d6b4c5d..0000000
Binary files a/src_3/data/MNIST/raw/train-labels-idx1-ubyte and /dev/null differ
diff --git a/src_3/data/MNIST/raw/train-labels-idx1-ubyte.gz b/src_3/data/MNIST/raw/train-labels-idx1-ubyte.gz
deleted file mode 100644
index 707a576..0000000
Binary files a/src_3/data/MNIST/raw/train-labels-idx1-ubyte.gz and /dev/null differ
diff --git a/src_3/data_loaders.py b/src_3/data_loaders.py
index 4033736..42b3323 100644
--- a/src_3/data_loaders.py
+++ b/src_3/data_loaders.py
@@ -46,6 +46,7 @@ class PecanSlice(Dataset):
 
 def make_train_data(config, train_data, appliance, window_length, train_buildings):
     root_path = "/home/Alfredo/input/1min_real_"
+    root_path = "/Users/alfredorodriguez/Desktop/input/1min_real_"
     train_dataset = PecanStreetDataset(str(root_path) + str(train_data) + "2019.csv", appliance, window_length,
                                              train_buildings)
 
@@ -56,7 +57,7 @@ def make_train_data(config, train_data, appliance, window_length, train_building
     train_dataset = PecanSlice(train_dataset[0], train_dataset[1])
 
     train_loader = DataLoader(dataset=train_dataset,
-                              batch_size=config.batch_size,
+                              batch_size=config['batch_size'],
                               shuffle=False,
                               num_workers=0)
 
@@ -65,7 +66,7 @@ def make_train_data(config, train_data, appliance, window_length, train_building
 
 def make_test_val_data(config, test_data, appliance, window_length, test_buildings):
     root_path = "/home/Alfredo/input/1min_real_"
-
+    root_path = "/Users/alfredorodriguez/Desktop/input/1min_real_"
     test_validation_dataset = PecanStreetDataset(str(root_path) + str(test_data) + "2019.csv", appliance, window_length,
                                                 test_buildings)
     seq_std = test_validation_dataset.y_std
